### PR TITLE
add CUDA_TOOKIT_ROOT_DIR as tensorrt detect dir

### PR DESCRIPTION
**Pull request by irexyc**
## Motivation

some user may copy the tensorrt library to cuda dir like /usr/local/cuda, so add ${CUDA_TOOLKIT_ROOT_DIR} as a search path of tensorrt. (#324) 

## Modification

- [x] cmake/modules/FindTENSORRT.cmake

### CODE REVIEW

**Review by tehkillerbee — APPROVED** [review]
Looks good to me!

File: cmake/modules/FindTENSORRT.cmake [hunk 1]
```diff
@@ -9,22 +9,24 @@ endif()
...
 foreach(__component ${__TENSORRT_LIB_COMPONENTS})
     find_library(
         __component_path ${__component}
-        HINTS ${TENSORRT_DIR}
+        HINTS ${TENSORRT_DIR} ${CUDA_TOOLKIT_ROOT_DIR}
         PATH_SUFFIXES lib lib64 lib/x64)
     if (NOT __component_path)
-        message(FATAL_ERROR "Cannot find TensorRT lib ${__component}, "
+        message(FATAL_ERROR "Cannot find TensorRT lib ${__component} in "
+            "CUDNN_DIR: ${CUDNN_DIR} or CUDA_TOOLKIT_ROOT_DIR: ${CUDA_TOOLKIT_ROOT_DIR}, "
```
Comment (tehkillerbee) [inline cmake/modules/FindTENSORRT.cmake:29]: By the way, shouldn't we reference TENSORRT_DIR instead of CUDNN_DIR here?

Comment (irexyc) [reply cmake/modules/FindTENSORRT.cmake]: Sorry, it is a typo. 

Fixed now.

Comment (tehkillerbee) [reply cmake/modules/FindTENSORRT.cmake]: @irexyc Ok! Approved.

**Review by lzhangzz — APPROVED** [review]

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**